src/commands/moderation/dehoist.js

```diff
@@ -0,0 +1,21 @@
+const Command = require("../../structures/Command");
+const Constants = require(`../../utility/Constants`);
+
+module.exports = class extends Command {
+    constructor(...args) {
+        super(...args, {
+            description: "Check if any members of the server has special characters in their username.",
```

Comment:
`Check if any members of the server have any special characters in their username.`